Tests/Features/resource_api_phpcr_odm.feature

```diff
@@ -90,3 +90,30 @@ Feature: PHPCR-ODM resource repository
                 }
             }
             """
+    @doNow
+    Scenario: Rename a PHPCR-ODM resource
+        Given there exists a "Article" document at "/cmf/articles/foo":
+            | title | Article 1 |
+            | body | This is my article |
+        Then I set header "Content-Type" with value "application/json"
+        When I send a PATCH request to "/api/phpcrodm_repo/foo" with body:
+            """
+            {"node_name": "foo-bar"}
+            """
+        Then the response code should be 200
+        When I send a GET request to "/api/phpcrodm_repo/foo-bar"
+        Then the response code should be 200
+        And the response should contain json:
```

Comment:
Imo, this is not how BDD should be done. We're way to close to the implementation at this point. I would rather suggest:
```
Scenario: Moving a PHPCR-ODM resource
    Given there exists an "Article" document at "/cmf/articles/hello-world":
        | title | Hi! |
        | body | Welcome to this great document! |
    When I send a PATCH request to "/api/phpcrodm_repo/hello-world" with body:
        """
        {"node_name": "/hello-world-1"}
        """
    Then the response code should be 200
    And there should be an "Article" document at "/cmf/articles/hello-world-1"
    And there should not be an "Article" document at "/cmf/articles/hello-world"
```